src/Gd/Image.php

```diff
@@ -727,6 +727,18 @@ private function saveOrOutput(Format $format, array $options, $filename = null)
                 break;
         }
 
+        if (isset($options['resolution-units']) && isset($options['resolution-x']) && function_exists('imageresolution')) {
+            $resolution_x = $options['resolution-x'];
+            $resolution_y = isset($options['resolution-y']) ? $options['resolution-y'] : $resolution_x;
+
+            if ($options['resolution-units'] === ImageInterface::RESOLUTION_PIXELSPERCENTIMETER) {
+                $resolution_x = $resolution_x * ImageInterface::RESOLUTION_PPC_TO_PPI_MULTIPLIER;
+                $resolution_y = $resolution_y * ImageInterface::RESOLUTION_PPC_TO_PPI_MULTIPLIER;
+            }
+
+            imageresolution($this->resource, $resolution_x, $resolution_y);
```

Comment:
```suggestion
            $resolutionX = $options['resolution-x'];
            $resolutionY = isset($options['resolution-y']) ? $options['resolution-y'] : $resolutionX;

            if ($options['resolution-units'] === ImageInterface::RESOLUTION_PIXELSPERCENTIMETER) {
                $resolutionX *= ImageInterface::RESOLUTION_PPC_TO_PPI_MULTIPLIER;
                $resolutionY *= ImageInterface::RESOLUTION_PPC_TO_PPI_MULTIPLIER;
            }

            imageresolution($this->resource, $resolutionX, $resolutionY);
```

I think the coding style of Imagine is to use camel case variable names (but I’m not sure ☺️)

Reply:
You're right. Thanks for your review.